docs/open-source-tizen/about/work-flow.md

```diff
@@ -1,136 +1,110 @@
+  - Approving submissions to OBS.
+  - Performing smoke tests on the resulting images and transferring them to the release area for QA engineers to perform further tests.
 
-- Approve submission to OBS.
-- Perform smoke-test of the resulting image and then transfer to the release area for QA engineers to perform further test.
+- **QA engineers** are responsible for:
 
-### QA Engineers
+  - Performing thorough integration and verification of the image to eliminate regressions and bugs.
 
-QA Engineers' responsibility is:
+## Package Development Workflow
 
-- Perform thorough integration and verification of the image to eliminate regressions and bugs.
+The package development workflow is described in the following procedure and shown in the following figure.
 
+**Figure: Package development workflow**
 
-
-### Package Development Work Flow
-
-The package development workflow is described in the following procedure and shown in the figure below:
-
-![img](https://source.tizen.org/sites/default/files/users/user-1132/800px-tizen-work-flow.png)
+![Package development workflow](media/800px-tizen-work-flow.png)
 
 1. Developers set up the development environment and install the development tools.
-
-2. Developers clone the source code, do required development, and perform local verification through local build.
-
-3. Developers submit patches to the Gerrit for all stakeholders to review.
-
-4. Tizen back-end service and reviewers verify the patches through automated and manual testing, respectively, and then vote -1, 0, or +1 based on the quality of the patches.
-
-   - Automated testing: Tizen back-end service automatically submits patches to the OBS to perform remote build and then posts the test results back to the Gerrit.
-   - Manual testing: Testers verify the patches manually and then publish comments on the Gerrit.
-
-5. Maintainers approve the patches (Code-Review +2) after they pass verifications (Verified +1) and then merge code changes to the Gerrit repository.
-
-6. Maintainers/Developers submit packages to the OBS by using **gbs submit** command.
-
-   **Note:** It's best practice to always have a maintainer role for every Tizen project, and maintainers should submit pakcages to the OBS as soon as all merged packages are ready for submission in order to prevent possible omission.
-
-7. Tizen back-end service activates pre-release and normal release processes at the same time. During the pre-release process, Tizen images that incoporates specific packages are created and submitted to release engineers to review.
-
-8. Release engineers accept or reject submissions based on the quality of the packages. For accepted submissions, the corresponding source code will be merged into the OBS repository, meanwhile, the normal release process takes over and publishes repos together with Tizen images.
+1. Developers clone the source code, do required development, and perform local verification through a local build.
```

Comment:
This list numbering issue is something we have queried separately.
If we keep the running numbering, it will cause extra work when the topic needs to be updated, a new step is added to the list, and the number for every single step needs to be updated manually to keep the content readable.
For this reason, we have consistently changed all step lists to use only number 1.
We can change this back and put running numbers to all step lists. But it will cause extra work in the future, so are you absolutely sure that you want this?

Reply:
Don't change lists that you already changed. I hope you don't change the ordered list from the next work when there is no problem.

Reply:
OK, the rule is then to number the lists manually, and to keep the numbering up-to-date during updates to avoid confusion.
We will follow this rule in the future, but will not apply it to this patch of topics.